modules/MySettings/TabController.php

```diff
@@ -195,48 +210,10 @@ public function get_user_tabs(&$user, $type='display')
         return array();
     }
 
-    public function get_unset_tabs($user)
```

Comment:
We wouldn't be able to remove public functions in hotfix/hotfix-7.10.x even if we're not using them anymore since it would break backwards compatibility.

Reply:
@Dillon-Brown Can I just base this on 7.11.x then?

Reply:
While not an LTS 7.11.x still has to follow semVer (Where possible anyway). If you wanted you could base it to develop which we'd have no problem merging in but otherwise will need to be made backwards compatible.

Reply:
@Dillon-Brown changing base to be `develop`